utils.js

```diff
@@ -75,6 +75,18 @@ async function checkResultAndReturnUrl(
   }
 }
 
+function maybeProcessColor(color?: string | number) {
+  if (color == null || typeof color === 'number') {
+    return color;
```

Comment:
what happen if put a wrong number here?

Reply:
it would render a weird color based on that number I think, but with the ts / flow types it should not allow passing a number directly. I added the number check here because if for some reason the color is processed twice it returns undefined.

Reply:
I'm ok with that, but I prefer showing invalid color warning from a native handled exception